clients/cli/src/config.rs

```diff
@@ -57,8 +57,8 @@ impl Config {
                 wallet_manager,
             ),
         }
-        .ok()
-        .map(Arc::from);
```

Comment:
Swallowing the error here resulted in a strange obfuscated error result while debugging an issue with the version bump. Mapping the error to a string instead so it can be read in the future.